strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/test/java/org/carlspring/strongbox/authentication/api/impl/ldap/LdapAuthenticationProviderTest.java

```diff
@@ -105,40 +87,23 @@ public void embeddedLdapServerRegistersExpectedAuthenticationProvider()
     @Test(expected = BadCredentialsException.class)
     public void ldapAuthenticationProviderShouldInvalidedOnWrongPassword()
     {
-        try
-        {
-            Class<?> loadClass = Thread.currentThread().getContextClassLoader().loadClass("org.carlspring.strongbox.authentication.api.impl.xml.DefaultAuthenticator");
-        }
-        catch (ClassNotFoundException e)
-        {
-            e.printStackTrace();
-        }
-        LdapAuthenticationProvider LdapAuthenticationProvider = getLdapAuthenticationProvider();
-        Authentication authentication = LdapAuthenticationProvider.authenticate(
+        LdapAuthenticationProvider LdapAuthenticationProvider = appCtx.getBean(LdapAuthenticationProvider.class);
+        LdapAuthenticationProvider.authenticate(
                 new UsernamePasswordAuthenticationToken("przemyslaw.fusik", "not-a-password"));
     }
 
-    protected LdapAuthenticationProvider getLdapAuthenticationProvider()
-    {
-        Iterator<Authenticator> i = authenticatorsRegistry.iterator();
-        i.next();
-        i.next();
-        LdapAuthenticationProvider LdapAuthenticationProvider = (org.springframework.security.ldap.authentication.LdapAuthenticationProvider) i.next().getAuthenticationProvider();
-        return LdapAuthenticationProvider;
-    }
-
-    @Test
-    public void registryShouldContainLdapAuthenticator()
+    @Override
+    protected Resource getAuthenticationConfigurationResource()
+            throws IOException
     {
-        Assert.assertTrue(StreamSupport.stream(authenticatorsRegistry.spliterator(), false).filter(
-                authenticator -> authenticator instanceof LdapAuthenticator).findFirst().isPresent());
+        return ConfigurationResourceResolver.getConfigurationResource("authentication.providers.xml",
+                                                                      "etc/conf/ldap-strongbox-authentication-providers.xml");
```

Comment:
This is just a special strongbox-authentication-providers.xml version for LDAP test. I'll rename it to strongbox-authentication-providers.xml. It's a residue of previous version.